### ISSUE TITLE
Training for a video with static background

### ISSUE THREAD

**Issue by shngt** [apple/ml-neuman#73]
It seems that the preprocessing fails for custom videos with a static background because colmap is unable to perform scene reconstruction. (ERROR: failed to create sparse model) Do you have any tips for handling videos like this?

Thank you very much for this project!

**Comment by jiangwei221** [comment]
No, our preprocessing system and our background NeRF can't handle such videos.
